schema/2.0.0/corpora_schema.md

```diff
@@ -145,13 +145,13 @@ The following ontology dependencies are *pinned* for this version of the schema.
 
 cellxgene requires ENSEMBL gene identifiers to ensure that all datasets it stores measure the same features and can therefore be integrated.
 
-The following gene annotation dependencies are *pinned* for this version of the schema.
+The following gene annotation dependencies are *pinned* for this version of the schema. **Cells from any organism are allowed as long as orthologs from these annotations are used.**
```

Comment:
I'm wondering whether this should be moved to the Requirements section where we state:

* **Organisms**. Data MUST be either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal.

Reply:
I think it would be would to clarify it this early. It's the first occurrence of a list of organisms, so it seems appropriate to let the user know that we are not restricting cells to only come from these organisms